test/unit/specs/components/wallet/__snapshots__/PageSend.spec.js.snap

```diff
@@ -1356,7 +1356,7 @@ exports[`PageSend should show bech32 error when address length is too long 1`] =
                 </h3>
                  
                 <h2>
-                  1,000.0000…
+                  0.0000…
```

Comment:
This looks wrong.

Reply:
Page send uses `fermions` and `mycoins`, so (i think ?) it's expected

Reply:
(the balance header uses `stake`)

Reply:
updated